htdocs/eventorganization/conferenceorbooth_list.php

```diff
@@ -360,6 +362,15 @@ function init_myfunc()
 	}
 	print '</td></tr>';
 
+	// Link to the vote/register page
+	print '<tr><td>'.$langs->trans("RegisterPage").'</td><td>';
+	$encodedid = dol_encode($project->id, $dolibarr_main_instance_unique_id);
+	$linkregister = $dolibarr_main_url_root.'/public/project/index.php?id='.$encodedid;
+	$encodedsecurekey = dol_encode($conf->global->EVENTORGANIZATION_SECUREKEY.$encodedid, $dolibarr_main_instance_unique_id);
```

Comment:
Do not use dol_encode to build the securekey.
Use 
dol_hash($conf->global->EVENTORGANIZATION_SECUREKEY.'conferenceofbotth'.$project->id, 2);